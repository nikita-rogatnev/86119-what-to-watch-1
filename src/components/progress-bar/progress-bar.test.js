import React from "react";
import Enzyme, {shallow} from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import toJson from "enzyme-to-json";

import ProgressBar from "./progress-bar";

Enzyme.configure({adapter: new Adapter()});

it(`ProgressBar renders correctly`, () => {
  const tree = shallow(<ProgressBar
    progress="10"
  />);

  expect(toJson(tree)).toMatchSnapshot();
});
