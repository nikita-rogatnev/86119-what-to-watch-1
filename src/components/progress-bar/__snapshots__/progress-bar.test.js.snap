// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar renders correctly 1`] = `
<progress
  className="player__progress"
  max="100"
  onClick={[Function]}
  value="10"
/>
`;
